src/javascript/app_2/Stores/Modules/Portfolio/portfolio-store.js

```diff
@@ -76,6 +76,11 @@ export default class PortfolioStore extends BaseStore {
         }
     }
 
+    @action.bound
```

Comment:
Can this be a `@computed`? (since it's not modifying the state?)

Reply:
@OskarAhl In fact I think it doesn't have to have a Mobx decorator at all, because `@computed` is for getter or property, but this is a function, and not an action.
Thanks. Will remove the decorator.